### PR TITLE
[MXNET-1086] added sub and mul to ONNX->TensorRT conversion

### PR DESCRIPTION
**Pull request by bermanmaxim**
## Description ##
Added the sub/mul operators to the MXnet -> ONNX conversion, for usage in the ONNX -> tensorRT conversion.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [x] Added ConvertElementwiseSub in nnvm_to_onnx-inl.h / nnvm_to_onnx.cc
- [x] Added ConvertElementwiseMul in nnvm_to_onnx-inl.h / nnvm_to_onnx.cc
- [x] Minor: fixed typo in CMakelists

## Comments ##
- The implementation directly follows ConvertElementwiseAdd.
- Fixes a bug since these missing operators were registered as TRTcompatible even before this commit.

### CODE REVIEW

**Comment by KellenSunderland** [comment]
So I was going to point out that we'd need to update the selector to add support for the mul / sub but it seems like that's already there.  This may actually fix model conversions for models that have those ops.  LGTM, any thoughts from your side @Caenorst?

https://github.com/apache/incubator-mxnet/blob/master/src/operator/subgraph/tensorrt/tensorrt-inl.h#L93

**Comment by Caenorst** [comment]
Yes, that was a mistake on my side.
The PR looks fine to me.

**Comment by KellenSunderland** [comment]
Test looks good, thanks for adding it.

File: tests/python/tensorrt/test_ops.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        executor = trt_sym.simple_bind(ctx=mx.gpu(), a=shape, b=shape,
+                                       grad_req='null', force_rebind=True)
+        y_trt_fp32 = executor.forward(is_train=False, a=a_data, b=b_data)
+        assert_almost_equal(y[0].asnumpy(), y_trt[0].asnumpy(), 1e-1, 1e-2)
+        assert_almost_equal(y[0].asnumpy(), y_trt_fp32[0].asnumpy(), 1e-4, 1e-4)
+    finally:
+        mx.contrib.tensorrt.set_use_fp16(original_precision_value)
+
+
+def test_elementwise():
```
Comment (marcoabreu) [inline tests/python/tensorrt/test_ops.py]: with_seed missing

**Pull request merged**